ipwb/util.py

```diff
@@ -33,7 +33,7 @@
 # or '/ip4/{ipaddress}/tcp/{port}/http'
 # or '/ip6/{ipaddress}/tcp/{port}/http
 
-IPWBREPLAY_ADDRESS = 'localhost:2016'
+IPWBREPLAY_ADDRESS = '0.0.0.0:2016'
```

Comment:
The `0.0.0.0` is a "listen to all interfaces" address, it can't be resolved from the client side.

Reply:
If you want host parameter to be customizable, then add a special case to see if it is set to `0.0.0.0` then for all URI rewriting and FQDN URI purposes use `localhost` (or the value in the `proxy` parameter).